### ISSUE TITLE
Unable to unlock two datasets that are locked for Ingest 

### ISSUE THREAD

**Issue by jggautier** [IQSS/dataverse.harvard.edu#151]
I mentioned this in a conversation in the IQSS Slack last month but thought I'd record it here too so it can be looked into when possible.

Two datasets are locked for Ingest and when I tried in early March 2022 to unlock them using the API endpoint, they went back into a locked state:

- The [API endpoint for checking locks](https://guides.dataverse.org/en/5.10/api/native-api.html?highlight=locks#manage-locks-on-a-specific-dataset) reports that https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/HVQDFB has been locked since 11/21/2019, even though I thought I unlocked it in March 2022 and that it was relocked because the ingest process started again.

- The API endpoint also reports that https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/E64YAB is still locked after I tried to unlock it in March 2022.

**Comment by sbarbosadataverse** [comment]
@jggautier has this been pushed to dev to unlock?

**Comment by jggautier** [comment]
It has yeah. The development team checks this GitHub repo and @landreev and I chatted about it in Slack the first time I unlocked those two dataset.

**Comment by mreekie** [comment]
If we can't unlock these we can't publish the data.  
It's a blocker for normal operations of DV

**Comment by landreev** [comment]
OK, I'll take a look. I have no idea why these could not be unlocked via the API, but I can do it by hand in the database.

**Comment by landreev** [comment]
OK, both datasets are unlocked now. 
Once again, I can't tell why they failed to get unlocked earlier.

**Comment by jggautier** [comment]
Thanks @landreev.

I'll email the depositor of the dataset at https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/E64YAB to let them know that they can publish the dataset when they'd like to.

Last month I emailed the depositor of the other dataset, the one at https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/HVQDFB, to ask if the dataset should be removed, since it's possible that the depositor has already published the data in a different dataset I found. The depositor hasn't replied, but this unpublished dataset will be deleted anyway in the curation team's quarterly or annual checks if it remains unpublished for a certain length of time.
